docs/user/TerraformAzureAuthentication.md

```diff
@@ -4,15 +4,15 @@ In order to create and destroy Azure resources on your behalf, Terraform needs a
 
 Your Service Principal or Managed Identity in the Azure subscription requires a **"Contributor"** role to create Azure resources. Follow these links for more information on how to create and retrieve the values to configure Terraform access to Azure. 
 
-- [How to create a Service Principal](./AzureHelpTopics.md#service-principal-using-azurecli)
+- [How to create a Service Principal](./AzureHelpTopics.md#service-principal-using-azurecli) | See Using a Service Principal Below
```

Comment:
Link should go to "https://github.com/andybouts/viya4-iac-azure/blob/main/docs/user/AzureHelpTopics.md#how-to-create-service-principal-using-azurecli"

Reply:
I need to understand this more.  That link is to my fork, which may get deleted once the PR is completed.  What's wrong with a relative link in the tree?  I just re-tested the relative link and it works as expected.  Can you please clarify?

Reply:
The relative is fine. It's the name of the section that changed 
what is there "#service-principal-using-azurecli" 
what it should be "#how-to-create-service-principal-using-azurecli"